include/grpc++/server_builder.h

```diff
@@ -71,7 +71,13 @@ class ServerBuilder {
   ServerBuilder();
   ~ServerBuilder();
 
-  enum SyncServerOption { NUM_CQS, MIN_POLLERS, MAX_POLLERS, CQ_TIMEOUT_MSEC };
+  /// Options for synchronous servers.
+  enum SyncServerOption {
+    NUM_CQS,         // Number of completion queues.
```

Comment:
Are you sure with doxygen syntax for enum values? https://stackoverflow.com/a/13756348/61505